data/json/recipes/other/vehicles.json

```
@@ -1866,8 +1866,9 @@
     "difficulty": 1,
     "autolearn": true,
     "time": "50 m",
```

Also time adjustment related to reduced amount of material being printed. :>
```suggestion
    "time": "30 m",
```